runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -7679,17 +7881,32 @@ int post_fma_ct(c_nodeid_t* locale_v, gni_post_descriptor_t* post_desc)
     PERFSTATS_ADD(rcvd_bytes, post_desc->length);
 
   {
-    gni_ct_put_post_descriptor_t* pdc;
-    int i;
+    if (post_desc->type == GNI_POST_FMA_PUT) {
+      gni_ct_put_post_descriptor_t* pdc;
+      int i;
 
-    for (pdc = post_desc->next_descr, i = 1;
-         pdc != NULL;
-         pdc = pdc->next_descr, i++) {
-      pdc->ep_hndl = cd->remote_eps[locale_v[i]];
-      if (post_desc->type == GNI_POST_FMA_PUT)
-        PERFSTATS_ADD(sent_bytes, pdc->length);
-      else
-        PERFSTATS_ADD(rcvd_bytes, pdc->length);
+      for (pdc = post_desc->next_descr, i = 1;
+           pdc != NULL;
+           pdc = pdc->next_descr, i++) {
+        pdc->ep_hndl = cd->remote_eps[locale_v[i]];
+        if (post_desc->type == GNI_POST_FMA_PUT)
+          PERFSTATS_ADD(sent_bytes, pdc->length);
```

Comment:
I think you can just remove the if-stmt here and do the `PERFSTATS_ADD(sent_bytes, ...)` only.  This was originally here to cover the case where we had chained GETs as well as PUTs, but we don't have chained GETs now and if we add them in the future we can update this.